app/src/main/res/layout-sw600dp-port/topic_lessons_title.xml

```diff
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<layout xmlns:android="http://schemas.android.com/apk/res/android">
-
-  <data>
-
-    <import type="android.view.View" />
-
-    <variable
-      name="viewModel"
-      type="org.oppia.android.app.home.recentlyplayed.SectionTitleViewModel" />
-  </data>
-
-  <FrameLayout
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:layout_marginTop="32dp">
-
```

Comment:
Maybe you shift this `layout_marginTop` to the text view and use `topic_lessons_title_margin_top` as its value.

Reply:
By shifting ### layout_marginTop to text view. Its layout chnges as shown in screenshot below.
![Screenshot 2022-01-16 101247](https://user-images.githubusercontent.com/89182224/149647663-61f035a5-3098-4c14-8f05-e00f11c2eb6f.png)
So,I just changes ### 32dp with ### topic_lessons_title_margin_top

Reply:
Okay then let it be